src/test/java/gift/repository/UserRepositoryTest.java

```diff
@@ -0,0 +1,48 @@
+package gift.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import gift.entity.User;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = Replace.NONE)
+class UserRepositoryTest {
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Test
+    @DisplayName("save 테스트")
+    public void saveTest() {
```

Comment:
junit5부터는 package-private이 가능합니다.
```suggestion
    void saveTest() {
```

Reply:
네 해당 방식으로 수정하였습니다